libs/ledger-live-common/src/hw/actions/startExchange.ts

```diff
@@ -129,38 +129,20 @@ export const createAction = (
           appName: "Exchange",
         };
       }
-      if (!exchange || !mainFromAccount || !mainToAccount) {
-        return {
-          appName: "Exchange",
-          requireLatestFirmware,
-        };
-      } else {
-        const shouldAddEthApp =
-          (mainFromAccount.currency.family === "evm" || mainToAccount.currency.family === "evm") &&
-          mainFromAccount.currency.managerAppName !== "Ethereum" &&
-          mainToAccount.currency.managerAppName !== "Ethereum";
```

Comment:
Looks like you removed a recent fix I did here, any reason we removed it ?

Reply:
Hello. Yes, I have removed that and just added this part:
`dependencies.push({ appName: mainFromAccount?.currency?.managerAppName });`

It makes the code easier to read, and adds any app as needed. @CremaFR tested it for swaps, and it works well.

Reply:
Yes but we need to keep the above logic to add ethereum in case of avax as it's need for swap

Reply:
Could you please add a comment next time to explain such changes? This was incredibly obscure, and no one could have guessed it without context. 
Even just linking the JIRA ticket would have been enough, as we often do for similar quirks in swaps https://github.com/LedgerHQ/ledger-live/blob/develop/apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/Form/index.tsx#L388

Reply:
Or put in a function like `addAvaxAppDep `

Reply:
And I'll make a PR to add PTX owner of this file too from now on

Reply:
Yes sure, the name of the variable and the logic was not that hard to follow IMO and I understand that you probably wanted to remove this but if something is obscure or doesn't make sense I usually find the author of the change and try to get answers before doing a change if none are provided in the source code
I'll keep this in mind if I end up in a similar situation, as I usually add context if needed for strange changes